### PR TITLE
AST-2473 - Improvement: Added compatibility for Astra 4.0.0

### PR DESCRIPTION
**Pull request by imnavanath**
- Astra customizer reset plugin should also clear Astra’s new admin settings, to get fresh setup not only for customizer but for admin part too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 **Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)  
 **Tags:** astra theme, customizer reset, reset astra customizer, reset astra theme  
 **Tested up to:** 6.1
```
Comment (imnavanath) [inline README.md:4]:
```suggestion
**Tested up to:** 6.1.1  
```

File: readme.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Contributors: brainstormforce
 Tags: astra theme, customizer reset, reset astra customizer, reset astra theme
 Tested up to: 6.1
```
Comment (imnavanath) [inline readme.txt:4]:
```suggestion
Tested up to: 6.1.1
```

File: class-astra-theme-customizer-reset.php [hunk 3]
```diff
@@ -116,6 +116,9 @@ public function ajax_customizer_reset() {
 				// Delete Global Color Palette and Typography Preset options.
 				delete_option( 'astra-typography-presets' );
 				delete_option( 'astra-color-palettes' );
+
```
Comment (nathanrodrigues2111) [inline class-astra-theme-customizer-reset.php:119]: Should we also reset mega menu options ? , Just a thought

File: class-astra-theme-customizer-reset.php [hunk 4]
```diff
@@ -116,6 +116,9 @@ public function ajax_customizer_reset() {
...
+				// Delete Astra's 4.0.0 admin setting options.
+				delete_option( 'astra_admin_settings' );
```
Comment (nathanrodrigues2111) [inline class-astra-theme-customizer-reset.php:121]: Should we also reset mega menu options ? , Just a thought

**Review by neerajmasai — APPROVED** [review]

File: class-astra-theme-customizer-reset.php (hunk 4 above)
Comment (imnavanath) [reply class-astra-theme-customizer-reset.php:121]: No actually, menu is totally different part. It comes under wp-core, megamenu options store in postmeta DB, so they can handle separately. Here we are only clearing theme's option table keys.

**Review by yssambare12 — APPROVED** [review]

**Pull request merged**